packages/neo-one-server-plugin-project/src/reset/index.ts

```diff
@@ -31,13 +31,15 @@ export const reset = (pluginManager: PluginManager, options: ResetTaskListOption
             const buildOptions: BuildTaskListOptions = {
               command: 'build',
               rootDir: project.rootDir,
+              browser: false,
```

Comment:
This is why it needs to be in the project config - we can't assume here that it's always false. E.g. think about what would happen if a user ran `neo-one build --browser` and then later `neo-one build --reset`